sdk/src/test/scala/com/horizen/account/performance/AccountSidechainNodeViewHolderPerfTest.scala

```diff
@@ -839,6 +840,75 @@ class AccountSidechainNodeViewHolderPerfTest
     }
   }
 
+  /* This test measures how long it takes to remove timed out transactions from the memory pool. */
+  @Test
+  //@Ignore
```

Comment:
uncomment `@Ignore` and double check the same for other performance tests.

Reply:
Done